### PR TITLE
data: SagarSharma2809

### PR DESCRIPTION
**Pull request by SagarSharma2809**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6945"></a>

### CODE REVIEW

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @SagarSharma2809, thank you for contributing :tada:

File: data/SagarSharma2809.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Sagar Sharma",
+  "type": "personal",
+  "bio": "**Front-End Developer** and **Open Source Contributor**. I share my journey through blogs"
```
Comment (khushalarora) [inline data/SagarSharma2809.json:4]: Please add a **'Comma'** at the end of this line :+1: 
```Suggestion
  "bio": "**Front-End Developer** and **Open Source Contributor**. I share my journey through blogs",
```

File: data/SagarSharma2809.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/sagar_2502/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/sagar-sharma-b8a894227/"
```
Comment (khushalarora) [inline data/SagarSharma2809.json:18]: Also, add a **'Comma'** at the end of this line :+1: 
```Suggestion
      "url": "https://www.linkedin.com/in/sagar-sharma-b8a894227/",
```

File: data/SagarSharma2809.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Check out my latest blogs on Hashnode",
+      "url": "https://sagarcoding.hashnode.dev/"
```
Comment (khushalarora) [inline data/SagarSharma2809.json:23]: Also, add a **'Comma'** at the end of this line :+1: 
```Suggestion
      "url": "https://sagarcoding.hashnode.dev/",
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @SagarSharma2809, thank you for contributing :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by SagarSharma2809** [comment]
thank you

**Pull request merged**